### ISSUE TITLE
result and epoch

### ISSUE THREAD

**Issue by Adithia99** [RoyWheels/stylegan2-ada#1]
the training was succesfull running.. but i dont know where the result , the model pkl, and how to define epoch ? its runing forever

**Comment by RoyWheels** [comment]
The results folder is set when you run the training command.

!python train.py --gpus=1 --data="/content/datasets/atari_384x256_dataset_raw" --min_h=2 --min_w=3 --res_log2=7 --snap=6 --cfg="auto" --mirror=1 --resume="latest" --metrics="none" <b>--outdir="/content/drive/My Drive/sg2ada_atari_results"</b>

You can see in the above training run using colab that I'm putting the results in my google drive. If the colab session expires , my latest PKL files will be safe.

By default the training runs for 25000 kimg. This is set in config, within the train.py file. I've never got to an end point myself. I'm producing models for creative uses so finishing training isn't really a thing I worry about.

**Comment by Adithia99** [comment]
Yeah thanks but i got little confuse in here , can u make the codes run forever (25k kimg)  , and got result per image in content result  in gcollab? 

Please i really need that give me the ipynb file :)
